nix/shell.nix

```diff
@@ -78,24 +79,6 @@ in {
 
   packages = lib.concatLists [ all-pkgs linux-pkgs ];
 
-  scripts.assemble-changelog = {
-    description = "Assembles the changelog for PACKAGE at VERSION";
-    exec = repoRoot.scripts."assemble-changelog.sh";
-    group = "changelog";
-  };
-
-  scripts.prepare-release = {
-    description = "Prepares to release PACKAGEs at VERSION";
-    exec = repoRoot.scripts."prepare-release.sh";
-    group = "changelog";
-  };
-
-  scripts.update-version = {
-    description = "Updates the version for PACKAGE to VERSION";
-    exec = repoRoot.scripts."update-version.sh";
-    group = "changelog";
-  };
```

Comment:
How about a `release-interactive` shell command that delegates to the shell script?